src/server/mod.rs

```diff
@@ -95,28 +99,31 @@ macro_rules! build_app {
             // image cache tester...
             //.service(web::resource("/v1/test").route(web::get().to(handlers::get_image)))
             // And finally the behavior necessary to satisfy Dockerflow
-            .service(web::scope("/").configure(dockerflow::service))
+            .service(web::scope("").configure(dockerflow::service))
     };
 }
 
 impl Server {
     /// initialize a new instance of the server from [Settings]
     pub async fn with_settings(mut settings: Settings) -> Result<dev::Server, HandlerError> {
-        let metrics = metrics_from_opts(&settings)?;
-        let mut raw_filter = HandlerResult::<AdmFilter>::from(&mut settings)?;
-        // try to update from the bucket if possible.
-        if raw_filter.is_cloud() {
-            raw_filter.update().await?
-        }
-        let filter = Arc::new(RwLock::new(raw_filter));
+        let metrics = Arc::new(metrics_from_opts(&settings)?);
         let req = reqwest::Client::builder()
```

Comment:
nit: tempted to rename this `client`.
We're building a `reqwest::Client`, that is passed to `.client()` and stored as `reqwest_client:`

Reply:
Agreed we can clean up this naming, I'll try to tackle it on later PR. I probably wouldn't use `client` now that we also have a `storage_client: cloud_storage::Client`. Maybe `reqwest` and `cloud_storage`?